src/seedu/addressbook/commands/SizeCommand.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.commands;
+
+/**
+ * Shows help instructions.
+ */
+public class SizeCommand extends Command {
+
+    public static final String COMMAND_WORD = "size";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Show the size of the address book.\n"
+            + "Example: " + COMMAND_WORD;
+
+    @Override
```

Comment:
Missing header comment. All non-trivial methods should have java doc format header comments.